src/constants/tooltips.js

```diff
@@ -50,5 +57,10 @@ export default {
     recoveryDestination: `The address your recovery transaction will send to.`,
     scan: 'The amount of addresses without transactions to scan before stopping the tool.',
     tokenAddress: 'The address of the smart contract of the token to recover. This is unique to each token, and is NOT your wallet address.',
+    apiKey: (coin) => {
+      if( coin === 'btc') {
```

Comment:
more spacing, please add a linter

Reply:
Fixed the spacing error, thanks for catching it! Yes we should add a linter, but I would prefer doing it outside the scope of this task. https://bitgoinc.atlassian.net/browse/BG-24127